File: palindromechecker.js
function checkIfPalindrome(string) {
  var reversed = string.split("").reverse().join(""); // Reverses the string - found on https://medium.freecodecamp.org/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb
  if ( string == reversed ) {
    // Runs this if the input is a palindrome
    console.log("\"" + string + "\"" + ' is a palindrome!');
  }
  else {
    // Runs this if the input is not a palindrome
    console.log("\"" + string + "\"" + ' is not a palindrome!');
  }
}

checkIfPalindrome("racecar"); //Palindrome
checkIfPalindrome("401Unauth"); //Not a palindrome

File: palindromeChecker.py
def palindromeCheck(string):
  reversed = string[::-1] #string[::-1] Reverses string. Found at https://www.geeksforgeeks.org/reverse-string-python-5-different-ways/
  if string == reversed:
    print(string + " is a palindrome!") #Output if it is a palindrome
  else:
    print(string + " is not a palindrome!") #Output if it is not a palindrome

palindromeCheck("palindrome") #Sample non-palindrome
palindromeCheck("racecar") #Sample palindrome
